### ISSUE TITLE
Quandidade de Assinaturas por compra

### ISSUE THREAD

**Issue by fgsants** [san0suke/pagseguro-assinaturas-rcs#16]
No momento não é possível adicionar o mesmo produto ao carrinho de novo. Como procedo para permitir isso?

**Comment by san0suke** [comment]
O plugin não permite. Eu fiz assim pra não ter que ficar fazendo vários tratamentos no carrinho.
Mas não é impossível de alguém programar este recurso.

**Comment by fgsants** [comment]
Pode me dar uma luz? vi que removendo as linhas 92-96 no PGA_Compra.class.php ele permite adicionar mais, porém só atualiza o valor total, não a quantidade...

PS.: estou planejando expandir o plugin posteriormente com mais funções. Ai te mando um pull se estiver interessado em incorporar.

**Comment by san0suke** [comment]
Sim, estou interessado em pull. Vou dar uma olhada depois e te falo

**Comment by san0suke** [comment]
No método pagamento da classe PGA_Assinaturas.class.php está a lógica de envio dos dados para o PagSeguro. O pagseguro recebe os dados e devolve uma URL, e através dela é feito o redirecionamento.
Está vendo o campo itemQuantity1 na linha 63? Tente bolar uma lógica no carrinho para dizer quantas unidades está tentando enviar.
